app/index.js

```diff
@@ -20,7 +20,7 @@ module.exports.start = function() {
 
     // Middlewares
     app.use([
-      express.static(path.join(__dirname, '/public')),
+      express.static(path.join(__dirname, 'public')),
```

Comment:
Here absolute path is needed because this is server-side code and `__dirname` returns path without trailing slash. So instead of `'/path/to/app' + '/public'` you'll get `'/path/to/app' + 'public'`